src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -38,7 +38,7 @@ private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
   // cake pattern to mock the scoverage report parser in tests
   scoverageReportParser: ScoverageReportParserAPI =>
 
-  import ScoverageSensorInternal._
+  import ScoverageSensorInternal._ // scalastyle:ignore Group imports
```

Comment:
So if you want to ignore this, I think you need to specify rule name, so in this case I believe that would be `scalastyle_ImportGroupingChecker`.
https://sonar.sonar-scala.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ImportGroupingChecker

Reply:
although I can see that you did the same in the Scoverage sensor and the ignore also seems to work when you specify the display name. 👍

Reply:
Hi, I did the same in the ScoverageSensor. But it was in this same PR _(and in the same commit)_ so it may fail for both :sweat_smile: (I haven't check it, yesterday I was in a hurry).

But I think it should work, because we have been using `// scalastyle:ignore LiteralArguments` all over the code base, and that's the display name - not the rule name.